beacon/README.md

```diff
@@ -0,0 +1,2 @@
+# Benchmarks, exploration, and analysis of Consensus
```

Comment:
Maybe at least link to our repo?

Reply:
I linked it in the main README. Should I add it here as well?

Reply:
Ah, I missed that; I think this should be enough then :+1: